pkg/controller/apimock/annotation_mock.go

```diff
@@ -4,11 +4,15 @@ import (
 	"context"
 	apirator "github.com/apirator/apirator/pkg/apis/apirator/v1alpha1"
 	v1 "k8s.io/api/core/v1"
+	"k8s.io/api/extensions/v1beta1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/types"
 )
 
 func (r *ReconcileAPIMock) processAnnotations(mock *apirator.APIMock) (updated bool, err error) {
+	var updateByIP = false
```

Comment:
You can use short var declaration in a single line:
Something like that:
`updateByIP, updateByPort, updateByIng := false, false, false`